### PR TITLE
graph-store: clean up old migrations, use some of the new +on methods

### PR DESCRIPTION
**Pull request by tacryt-socryp**

### CODE REVIEW

File: pkg/arvo/app/graph-store.hoon [hunk 1]
```diff
@@ -5,9 +5,9 @@
 |%
 +$  card  card:agent:gall
 +$  versioned-state
-  $%  [%0 network:zero:store]
-      [%1 network:zero:store]
-      [%2 network:zero:store]
+  $%  [%0 *]
+      [%1 *]
+      [%2 *]
```
Comment (liam-fitzgerald) [inline pkg/arvo/app/graph-store.hoon:10]: These are all pre breach, yeah?

File: pkg/arvo/mar/graph/update-0.hoon [hunk 2]
```diff
@@ -1,18 +0,0 @@
-/+  *graph-store
```
Comment (liam-fitzgerald) [inline pkg/arvo/mar/graph/update-0.hoon:1]: These marks are definitely not pre breach

**Review by liam-fitzgerald — COMMENTED** [review]
Some questions about migration and perf regressions. Else looks good so far

File: pkg/arvo/app/graph-push-hook.hoon [hunk 3]
```diff
@@ -316,28 +314,27 @@
...
+      (~(get by graph-to-mark.cache) resource)
     (get-mark:gra resource)
   ?~  mark
     [[%no %no %no] ~]
   =/  key  [u.mark (perm-mark-name perm)]
-  =/  perms-cached  (~(has by perm-marks.cache) key)
   =/  convert
-    ?:  perms-cached
-      (~(got by perm-marks.cache) key)
+    %+  fall
```
[the remark this answers is not in the document]
Comment (tacryt-socryp) [reply pkg/arvo/app/graph-push-hook.hoon:325]: I looked at the code. b is not evaluated unless a is sig.

File: pkg/arvo/app/graph-store.hoon (hunk 1 above)
Comment (tacryt-socryp) [reply pkg/arvo/app/graph-store.hoon:10]: %2 is not, but %0 and %1 are. Would you like me to add back %2?

**Comment by liam-fitzgerald** [comment]
Yeah please, I'm hesitant to drop support for anything that's not pre-breach

——
~hastuc-dibtux

https://urbit.org

**Comment by tacryt-socryp** [comment]
@liam-fitzgerald Only removed `%0` and `%1` migrations now. Reverted the controversial bits. This should be good to go.

**Review by liam-fitzgerald — APPROVED** [review]

**Pull request merged**